driver_test.go

```diff
@@ -1793,7 +1793,7 @@ func TestOpenWithTransport(t *testing.T) {
 
 	// Test that transport override also works in insecure mode
 	countingTransport.requests = 0
-	config.InsecureMode = true
+	config.DisableOCSPChecks = true
```

Comment:
I think we should have at least two tests (also for InsecureMode)

Reply:
Done.